src/ui/components/utils/input_widget.rs

```diff
@@ -265,14 +258,9 @@ impl<T: Clone> Component for InputWidget<T> {
                     self.input.backspace(1, &mut self.input_listener);
                 }
                 Key::Tab => {
-                    if self.config.borrow().storage.channels {
-                        if let Some(suggestion) = &self.suggestion {
-                            self.input.update(
-                                suggestion,
-                                suggestion.len(),
-                                &mut self.input_listener,
-                            );
-                        }
+                    if let Some(suggestion) = &self.suggestion {
```

Comment:
I believe you had already made this change, perhaps you need to rebase your branch with main?